config/spryk/templates/Zed/Business/MessageBroker/MessageHandler.php.twig

```diff
@@ -28,12 +28,12 @@ class {{ className | classNameShort }} implements {{ interfaceName | classNameSh
     }
 
     /**
-     * @var \Spryker\Shared\Kernel\Transfer\TransferInterface $newMessageTransfer
-     * @var \Spryker\Shared\Kernel\Transfer\TransferInterface|null $previousMessageTransfer
+     * @param \Spryker\Shared\Kernel\Transfer\TransferInterface $newMessageTransfer
+     * @param \Spryker\Shared\Kernel\Transfer\TransferInterface|null $previousMessageTransfer
      */
     public function sendMessage($newMessageTransfer, $previousMessageTransfer = null): void
```

Comment:
what is a use case for `$previousMessageTransfer` and copied MessageAttributes fro prev message? What if `$newMessageTransfer` has not empty MessageAttributes?

Reply:
When the received message results in a sub-sequent message it becomes handy as everything that is in your previous messageAttributes is just forwarded and you don't need to take care of it.

Reply:
anyway, it will require changing the `emitter` property. But OK, it's not a problem to do after copying attributes.